python.d/ipfs.chart.py

```diff
@@ -11,6 +11,7 @@
 # update_every = 2
 priority = 60000
 retries = 60
+pinapi = 0
```

Comment:
move it to the `__init__` (like `self.do_pinapi = self.configuration.get('pinapi')`)